### DISCUSSION TITLE
OpenJ9 support / more exact wording

### DISCUSSION THREAD

**Discussion by bmarwell** [microsoft/gctoolkit#20] [Ideas]
Hi everyone!

Thanks for sharing such a great tool! I would love to try it, but it seems that it is only compatible with HotSpot JVMs.
Consider this sentence from your README:

> GCToolKit is a set of libraries for analyzing Java garbage collection (GC) log files

It should probably be:

> GCToolKit is a set of libraries for analyzing **_Hotspot_** Java garbage collection (GC) log files

I know MS' JVM is yet another hotspot distribution like 99% of the other JVMs out there, but [OpenJ9](https://www.eclipse.org/openj9/) does have a significant use case and is typically used in conjunction with [OpenLiberty](https://github.com/OpenLiberty/open-liberty).

Many thanks,
\- Ben

**Comment by karianna** [comment]
Now #21 - Pull requests welcome!
